include/crocoddyl/core/integrator/rk4.hpp

```diff
@@ -123,24 +136,32 @@ struct IntegratedActionDataRK4Tpl : public ActionDataAbstractTpl<_Scalar> {
   virtual ~IntegratedActionDataRK4Tpl() {}
 
   VectorXs dx;
+  std::vector<VectorXs> u_diff;
```

Comment:
I believe it is better to use `std::array` as we are supporting c++11>. But, this might need to include extra code for binding them (similar to this code: https://github.com/loco-3d/crocoddyl/blob/devel/bindings/python/crocoddyl/utils/vector-converter.hpp#L117).

Feel free to ignore this request if you don't how to handle it.

Reply:
Since I am short on time I prefer to skip this as well.

Reply:
I would like to give a try on this topic (and in this PR) as it leads to an improvement in the code.
However, this will not block the merging of this PR. In worse case scenario, I will create an issue to track this needed change.

Please let me reopen this conversation (only for my own purpose)